modules/client/examples/01-tokenVoting-client/06-can-vote.ts

```diff
@@ -18,10 +18,9 @@ const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
 const client = new TokenVotingClient(contextPlugin);
 
 const voteParams: ICanVoteParams = {
-  address: "0x1234567890123456789012345678901234567890",
-  proposalId: "0x1234567890123456789012345678901234567890_0x1",
+  proposalId: "0x1234567890123456789012345678901234567890_0x0",
   pluginAddress: "0x1234567890123456789012345678901234567890",
```

Comment:
`pluginAddress` might be redundant after the change

Reply:
this was odd. changed it to match the interface